src/pal/src/exception/seh-unwind.cpp

```diff
@@ -96,30 +123,6 @@ static void WinContextToUnwindCursor(CONTEXT *winContext, unw_cursor_t *cursor)
     unw_set_reg(cursor, UNW_X86_64_R13, winContext->R13);
     unw_set_reg(cursor, UNW_X86_64_R14, winContext->R14);
     unw_set_reg(cursor, UNW_X86_64_R15, winContext->R15);
-#elif defined(_ARM_)
-    // Assuming that unw_set_reg() on cursor will point the cursor to the
-    // supposed stack frame is dangerous for libunwind-arm in Linux.
-    // It is because libunwind's unw_cursor_t has other data structure
-    // initialized by unw_init_local(), which are not updated by
-    // unw_set_reg().
-    unw_context_t context;
-    context.regs[0] = 0;
-    context.regs[1] = 0;
-    context.regs[2] = 0;
-    context.regs[3] = 0;
-    context.regs[4] = winContext->R4;
-    context.regs[5] = winContext->R5;
-    context.regs[6] = winContext->R6;
-    context.regs[7] = winContext->R7;
-    context.regs[8] = winContext->R8;
-    context.regs[9] = winContext->R9;
-    context.regs[10] = winContext->R10;
-    context.regs[11] = winContext->R11;
-    context.regs[12] = 0;
-    context.regs[13] = winContext->Sp;
-    context.regs[14] = winContext->Lr;
-    context.regs[15] = winContext->Pc;
-    unw_init_local(cursor, &context);
```

Comment:
``PAL_VirtualUnwind`` who invokes this function will invoke ``unw_init_local`` after this function returns.